core/common/src/main/java/alluxio/metrics/MetricsSystem.java

```diff
@@ -55,25 +55,17 @@
    * An enum of supported instance type.
    */
   public enum InstanceType {
-    MASTER("master"),
-    WORKER("worker"),
-    CLIENT("client"),
-    PROXY("proxy");
-
-    private String mValue;
+    MASTER,
+    WORKER,
+    CLUSTER,
+    CLIENT,
+    PROXY;
 
     /**
-     * Creates the instance type with value.
-     *
-     * @param value value of the instance type
+     * @return the lower case of the string value
      */
-    InstanceType(String value) {
-      mValue = value;
-    }
-
-    @Override
-    public String toString() {
-      return mValue;
+    public String getString() {
+      return this.toString().toLowerCase();
```

Comment:
How important is it to do this in lower case?

Reply:
This value will be part of the metric name, and for now the convention is to use lower case?